docs/quickstarts/nextjs.mdx

```diff
@@ -40,62 +68,66 @@ CLERK_SECRET_KEY={{secret}}
 
 </InjectKeys>
 
-### Mount `<ClerkProvider />`
-
-Add the [`<ClerkProvider />`][clerk-provider] wrapper to your application. The [`<ClerkProvider />`][clerk-provider] component wraps your Next.js application to provide active session and user context to Clerk's hooks and other components. A reasonable default approach is to have [`<ClerkProvider />`][clerk-provider] wrap the `<body/>` to enable the context to be accessible anywhere within the app.
+### Wrap your app in `<ClerkProvider />`
 
+The [`<ClerkProvider />`][clerk-provider] component provides active session and user context to Clerk's hooks and other components. For this project, it makes sense to wrap the `<html/>` element with `<ClerkProvider />`. This makes the active session and user context accessible anywhere within the app.

```

Comment:
We had specifically decided to move away from early education around opting into dynamic rendering when wrapping with the `ClerkProvider` right?